src/HierarchyLogEntry.h

```diff
@@ -38,20 +39,19 @@ class HierarchyLogEntry
     void notePeerWrite();
 
     /// Start recording total time spent communicating with peers
-    void startPeerClock();
+    void startPeerClock() { peeringTime_.resume(); }
```

Comment:
If we are touching/moving/removing all startPeerClock() calls, then I suggest renaming these methods to reduce chances of backporting bugs that incorrectly preserve some unmodified startPeerClock() calls (and to provide a visual proof that the PR has changed all the callers).

```suggestion
    void startPeering() { peeringTime_.resume(); }
```

... and stopPeering() to match, of course.

If you prefer, we can use resumePeering() and pausePeering(). Either way, the call names will mislead a bit when called for the first or second time...

Reply:
Ok, let's use startPeering()/stopPeering() names.